newrelic-agent/src/main/java/com/newrelic/agent/transport/apache/ApacheSSLManager.java

```diff
@@ -125,6 +56,9 @@ private static KeyStore getKeyStore(String caBundlePath)
                 while (is.available() > 0) {
                     try {
                         caCerts.add((X509Certificate) cf.generateCertificate(is));
+                    } catch (CertificateException e) {
+                        Agent.LOG.log(Level.SEVERE, "Failed to generate ca_bundle_path certificate. Verify the certificate format. For instance, is it this issue JDK-8208602 ?", e);
+                        break;
                     } catch (Throwable t) {
                         Agent.LOG.log(Level.SEVERE,
                                 "Unable to generate ca_bundle_path certificate. Will not process further certs.", t);
```

Comment:
Adding the new catch block above will prevent the code from getting into this catch block and logging `Unable to generate ca_bundle_path certificate. Will not process further certs` since the exception that was making it here was `CertificateException`.

```
2021-03-11T15:51:17,781-0800 [89136 1] com.newrelic ERROR: Unable to generate ca_bundle_path certificate. Will not process further certs.
java.security.cert.CertificateException: Could not parse certificate: java.io.IOException: Illegal header: -----BEGIN CERTIFICATE----- 
```

Why don't we get rid of the new catch and just merge the two log lines together in the old catch block:

```
} catch (Throwable t) {
                        Agent.LOG.log(Level.SEVERE,
                                "Unable to generate ca_bundle_path certificate. Please verify the certificate format. Will not process further certs.", t);
}
```

In general it's a terrible idea to `catch (Throwable t)` and catching a more specific exception (like you were doing with `CertificateException`) would be preferred but I don't know if `Throwable` is used here for a specific reason or not and I'd prefer to change as little behavior as possible around this logic.